### PR TITLE
Unit tests for urbansound8k

### PR DESCRIPTION
**Pull request by justinsalamon**

### CODE REVIEW

**Comment by justinsalamon** [comment]
@magdalenafuentes here are the missing unit tests for urbansound8k, I've also bumped the version to 0.1.0-rc2.

File: tests/datasets/test_urbansound8k.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+def test_load_audio():
+    dataset = urbansound8k.Dataset(TEST_DATA_HOME)
+    clip = dataset.clip("135776-2-0-49")
+    audio_path = clip.audio_path
+    audio, sr = urbansound8k.load_audio(audio_path)
+    assert sr == 44100
+    assert type(audio) is np.ndarray
+    assert len(audio.shape) == 1  # check audio is loaded as mono
+    assert audio.shape[0] == 176400  # Check audio duration in sampels is as expected
```
Comment (magdalenafuentes) [inline tests/datasets/test_urbansound8k.py]: What about reducing the size of the file to 1s? 😬😬😬

**Review by magdalenafuentes — APPROVED** [review]
Looks good! Just a minor comment and ready to go!

File: tests/datasets/test_urbansound8k.py (hunk 1 above)
Comment (justinsalamon) [reply tests/datasets/test_urbansound8k.py]: Good catch! I just trimmed it to 1sec and updated the test accordingly

**Pull request merged**